DotNetHowToSynonyms/DotNetHowToSynonyms/App.config

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
+  </startup>
+  <appSettings>
+    <add key="SearchServiceName" value="Put your search service name here" />
+    <add key="SearchServiceAdminApiKey" value="Put your primary or secondary API key here" />
+    <add key="SearchServiceQueryApiKey" value="Put your query API key here" />
+    <add key="ClientSettingsProvider.ServiceUri" value="" />
```

Comment:
This file has many unnecessary settings. Please use a copy of the App.config file in the DotNetHowTo project.